baseframe/filters.py

```diff
@@ -178,36 +179,37 @@ def cdata(text):
     return Markup('<![CDATA[' + text.replace(']]>', ']]]]><![CDATA[>') + ']]>')
 
 
-@baseframe.app_template_filter('shortdate')
-def shortdate(value):
+@baseframe.app_template_filter('date')
+def date_filter(value, fm='medium'):
```

Comment:
Call it `format` since you're simply passing this on. You will have to ignore a warning from PyFlakes.